src/lib/queries/reward/payReward.ts

```diff
@@ -44,7 +44,9 @@ export const usePayReward = () => {
   return useMutation({
     mutationFn: (rewardId: string) => payReward(wallet, rewardId),
     onSuccess: () => {
-      void queryClient.invalidateQueries({queryKey: queryKeys.rewards.all._def})
+      void queryClient.invalidateQueries({
+        queryKey: queryKeys.rewards.toPayout._def,
```

Comment:
I think every query that receives `rewardTxHash` should be invalidated as well (e.g. findings...)